### DISCUSSION TITLE
Too Many Requests

### DISCUSSION THREAD

**Discussion by arunika0** [YOURLS/YOURLS#3719] [Q&A]
Hi, any idea how this happens?

I tried to create a link but a notification appeared as shown. I've also looked for issues on GitHub but didn't find anything relevant

Or is there a way to limit the links created?

![Screenshot_2024-02-09-06-30-46-287_com android chrome](https://github.com/YOURLS/YOURLS/assets/40335008/ff7f8269-8289-486d-a7e3-e51266df5425)

**Comment by dgw** [comment]
The error most likely comes from https://github.com/YOURLS/YOURLS/blob/02278cd4eb6d2e5f28fb5416ca49d9388199f189/includes/functions.php#L668

Maybe your instance is reverse proxied but all remote clients appear to be on the same IP?

Logged-in users are not supposed to be throttled, too.

**Reply by arunika0** [reply]
okay thanks for the answer
